lib/merchant.rb

```diff
@@ -22,10 +24,21 @@ def invoices
     @invoices ||= @repository.find_invoices(id)
   end
 
-  def revenue
-    successful_invoice_items =  successful_invoices.map { |i| i.invoice_items }.flatten
-    successful_invoice_items.reduce(0) do |revenue, invoice_item|
-      revenue + (invoice_item.quantity * invoice_item.unit_price)
+  def revenue(date=nil)
```

Comment:
Assignment Branch Condition size for revenue is too high. [31.59/15]<br>Surrounding space missing in default value assignment.